music_assistant/config.yaml

```diff
@@ -1,6 +1,6 @@
-name: Music Assistant
+name: Music Assistant Server
 version: 2.0.7
-slug: music_assistant
+slug: music_assistant_server
```

Comment:
dont update the slug - this will cause the addon to be invalid for all existing installs!

Reply:
Yep don’t want that. Fixed.